### DISCUSSION TITLE
Scelta UPS NUT

### DISCUSSION THREAD

**Discussion by Simonv92** [indomus/forum#2987] [Hardware & Firmware]
Ciao a tutti! 
sto realizzando un acquario con controllo tramite Home Assistant. Vorrei integrare un UPS monitorato con NUT così da poter automatizzare lo spegnimento di alcuni carichi durante un blackout.
Ho visto diverse guide riguardo la configurazione di NUT su Raspberry Pi e l’integrazione in HA.
Ho provato a configurare un Riello idialog 1200 ma su HA vedo veramente pochi dati rispetto all’esempio trovato sul sito con il Tecnoware se non ricordo male.
Ho sbagliato qualcosa o la quantità di dati mostrata varia da modello a modello? Nel caso avreste un modello da consigliarmi intorno ai 1200/1500VA che permetta una buona integrazione tramite NUT?
Grazie mille
Simone

**Comment by indomus** [comment]
Leggi questo: https://indomus.it/guide/come-scegliere-un-ups-per-la-propria-domotica-e-altro/

**Reply by Simonv92** [reply]
Grazie mille per le info! È possibile capire a priori quali dati vengono esposti dall’UPS in caso di collegamento con NUT?

**Reply by indomus** [reply]
eh, dipende da ups a ups...

**Reply by Simonv92** [reply]
Ad esempio optando per Tecnoware la serie ERA ed EXA potrebbero esporre gli stessi dati?
